nixos/modules/services/networking/consul.nix

```diff
@@ -40,15 +29,6 @@ in
         '';
       };
 
-
-      webUi = mkOption {
```

Comment:
Instead of simply removing the `config.` options, I think it would be better to make use of them emit an explicit error message, that explains what to use instead (can be removed in the subsequent NixOS release).